### ISSUE TITLE
Spacer having a screen reading text while it's a styling SVG

### ISSUE THREAD

**Issue by Sboonny** [freeCodeCamp/freeCodeCamp#48700]
**Describe the issue**
In this file https://github.com/Sboonny/freeCodeCamp/blob/29f0897a8b2ca3176e7452c3d4da8fcff9c94723/client/src/assets/icons/spacer.tsx#L10-L28

You will notice that there is a screen reader text for spacer icon, which I think it's unnecessary because the icon doesn't provide additional context, but it's there for styling

I am wondering was there a reason for adding the screen reader text and not hiding it?

I found it while checking FormGroup in the setting page. If there is a reason for it, should we hide the one in "recieve Quincy's emails" toggle button in the setting page?

![image](https://user-images.githubusercontent.com/88248797/206727308-2a3ad5d5-bbc0-4927-be30-fc04dfee6c3f.png)

**Comment by bbsmooth** [comment]
Definitely, if an image is purely decorative then it should be hidden from screen readers completely, and in this specific case I would agree that the sr-only text should be removed. If there really is a situation where it would be beneficial for screen reader users to hear that there is a spacer icon then we could add a prop to trigger it. But the default should probably be not to have the sr-only text.

On a side note, the accessibility on those toggle buttons on the Settings page leaves much to be desired. I've been meaning to tackle those for quite a while now and this is a good reminder :slightly_smiling_face:

**Comment by Sboonny** [comment]
If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

**Comment by Endless8** [comment]
> **Describe the issue** In this file https://github.com/Sboonny/freeCodeCamp/blob/29f0897a8b2ca3176e7452c3d4da8fcff9c94723/client/src/assets/icons/spacer.tsx#L10-L28
> 
> You will notice that there is a screen reader text for spacer icon, which I think it's unnecessary because the icon doesn't provide additional context, but it's there for styling
> 
> I am wondering was there a reason for adding the screen reader text and not hiding it?
> 
> I found it while checking FormGroup in the setting page. If there is a reason for it, should we hide the one in "recieve Quincy's emails" toggle button in the setting page?
> 
> ![image](https://user-images.githubusercontent.com/88248797/206727308-2a3ad5d5-bbc0-4927-be30-fc04dfee6c3f.png)

We just need to remove the HTML text in the span element from the Spacer component?

**Comment by Sboonny** [comment]
and add `aria-hidden` 👍
